scripts/update-verification-metadata

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+cd ../
+rm gradle/verification-metadata.xml
+cp scripts/verification-metadata-template.xml gradle/verification-metadata.xml
+./gradlew --no-daemon --refresh-dependencies --write-verification-metadata sha256 assemble
```

Comment:
```suggestion
set -e

# remove the verification-metadata.xml file with old deps (if any), and replace it with the template
if [ -f ./gradle/verification-metadata.xml ]; then
  rm ./gradle/verification-metadata.xml
  cp ./scripts/verification-metadata-template.xml ./gradle/verification-metadata.xml
fi

# generate verification-metadata.xml
./gradlew --no-daemon --refresh-dependencies --write-verification-metadata sha256 assemble
```

better to fail ( via `set -e` ) and exit, if for some reason file cannot be removed or the template is missing